src/plugins/dashboard/public/application/legacy_app.js

```diff
@@ -230,7 +222,9 @@ export function initDashboardApp(app, deps) {
                   );
                   return new Promise(() => {});
                 } else {
-                  throw error;
+                  // E.g. a corrupt dashboard was detected (e.g. with invalid JSON properties)
```

Comment:
I'm just wondering if there are any implications of eliminating throwing an error here.

Reply:
The error we were discussing was throw this way, and that's why we got an empty dashboard. By changing this, the user is redirected to the listing page, and an error message is displayed. Of course we've to check for a case that requires an error thrown this way, but I do think this is legacy